config/configstore/service.go

```diff
@@ -81,63 +82,51 @@ func (s service) GenerateAccount() (config.Account, error) {
 		return nil, err
 	}
 
-	id, confirmations, err := s.idService.CreateIdentity(ctx, identity.RandomCentID())
+	DID, err := s.idFactory.CreateIdentity(ctx)
```

Comment:
should we call it `DID` or `did`?

Reply:
I like DID for method arguments and I extended that to internal variables within the method too. But I am open to suggestions.